packages/@webex/plugin-cc/src/services/core/aqm-reqs.ts

```diff
@@ -0,0 +1,314 @@
+import {Signal} from './Signal'; // TODO: remove and use Event Emitters after Adhwaith PR is merged
+import {Msg} from './GlobalTypes';
+import * as Err from './Err';
+import {HTTP_METHODS, WebexRequestPayload} from '../../types';
+import HttpRequest from './HttpRequest';
+import LoggerProxy from '../../logger-proxy';
+import {
+  CbRes,
+  Conf,
+  ConfEmpty,
+  EvtConf,
+  EvtRes,
+  Pending,
+  Req,
+  Res,
+  ResEmpty,
+  Timeout,
+} from './types';
+
+export const TIMEOUT_REQ = 20000;
+const TIMEOUT_EVT = TIMEOUT_REQ;
+
+export class AqmReqs {
+  private pendingRequests: Record<string, Pending> = {};
+  private pendingEvents: Record<string, Pending> = {};
+  // pendingNotifCancelrequest is used for handling request cancel events based on another action. When a promise can be resolved using multiple events.
```

Comment:
We do not require these comments. Please remove it

Reply:
addressed